From 8cca43926552c481479a1027130778025aba1c47 Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Wed, 3 Mar 2021 16:18:19 -0600
Subject: [PATCH 04/13] HSD #22012255070: crypto: intel_fcs: reset the min. and
 max. buffer size values

commit  a1ce9a448215127c9c46d2587ec4df78383c57bb from
https://github.com/altera-opensource/linux-socfpga.git

Reset the minimum and maximum encrypted buffer size value by removing the
8-bytes padding.

Signed-off-by: Richard Gong <richard.gong@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/crypto/intel_fcs.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/crypto/intel_fcs.c b/drivers/crypto/intel_fcs.c
index 13f221a90d0f..f5fb1da2110e 100644
--- a/drivers/crypto/intel_fcs.c
+++ b/drivers/crypto/intel_fcs.c
@@ -30,8 +30,8 @@
 
 #define DEC_MIN_SZ		72
 #define DEC_MAX_SZ		32712
-#define ENC_MIN_SZ		128
-#define ENC_MAX_SZ		32768
+#define ENC_MIN_SZ		120
+#define ENC_MAX_SZ		32760
 
 
 #define FCS_REQUEST_TIMEOUT (msecs_to_jiffies(SVC_FCS_REQUEST_TIMEOUT_MS))
-- 
2.17.1

